### ISSUE TITLE
[@angular-eslint/no-input-rename] Reports false error when using transform with function which accept 2 parameters

### ISSUE THREAD

**Issue by Wycza** [angular-eslint/angular-eslint#1861]
**Description and reproduction of the issue**

```TS
@Input({ transform: (val: unknown) => numberAttribute(val, 0) }) numberValue!: number;
```
throws the false error:

```TS
Input bindings should not be aliased (https://angular.dev/style-guide#style-05-13)
```

The problem is with the `fallbackValue` parameter of the `numberAttribute` function.

```
(alias) numberAttribute(value: unknown, fallbackValue?: number | undefined): number
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `angular-eslint`    | `18.0.1` |
| `@typescript-eslint/parser`        | `8.0.0-alpha.20` |
| `ESLint`                           | `9.3.0` |
| `node`                             | `22.2.0` |

```sh
Angular CLI: 18.0.2
Node: 22.2.0
Package Manager: npm 10.7.0
OS: win32 x64

Angular: 18.0.1
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1800.2
@angular-devkit/build-angular   18.0.2
@angular-devkit/core            18.0.2
@angular-devkit/schematics      18.0.2
@angular/cli                    18.0.2
@schematics/angular             18.0.2
rxjs                            7.8.1
typescript                      5.4.5
zone.js                         0.14.6

```
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by LayZeeDK** [comment]
I have the same issue.

Works with

```typescript
@Input({ transform: coerceNumberProperty })
count = 0;
```

Does not work with

```typescript
@Input({ transform: (x: unknown) => coerceNumberProperty(x, 0) })
count = 0;
```

or

```typescript
@Input({ transform: x => coerceNumberProperty(x, 0) })
count = 0;
```

| Package | Version |
| --- | --- |
| `@angular/core` | `16.2.12` |
| `@angular-eslint/eslint-plugin` | `16.1.0` |
| `eslint` | `8.46.0` |

**Comment by LayZeeDK** [comment]
Possibly related to #1446 and #1809.

**Comment by LayZeeDK** [comment]
@Wycza 

Please try `@angular-eslint/*` version 18.1.0 or newer, released after #1809 was merged.
